test/content-api-v1.js

```diff
@@ -0,0 +1,121 @@
+/* eslint-env mocha */
+'use strict';
+
+const request = require('supertest');
+
+require('should');
+
+const testServerPort = 3000;
+const HTTP_OK = 200;
+const HTTP_BAD_REQUEST = 400;
+
+const no_region = {document: '{\n  \"instanceId\" : \"i-aaaaaaaa\",\n  \"accountId\" : \"123456789012\",\n  \"imageId\" : \"ami-abcdeaf2\",\n  \"region\" : \"UK-Northeast-SouthWest-7\"\n }', signiture: '-----BEGIN PKCS7-----END PKCS7-----\n'};
+const no_instance = {document: '{\n  \"instanceId\" : \"i-a9aza\",\n  \"accountId\" : \"123456789012\",\n  \"imageId\" : \"ami-abcdeaf2\",\n  \"region\" : \"us-east-2\"\n }', signiture: '-----BEGIN PKCS7-----END PKCS7-----\n'};
+const no_account = {document: '{\n  \"instanceId\" : \"i-aaaaaaaa\",\n  \"accountId\" : \"abcdefg\",\n  \"imageId\" : \"ami-abcdeaf2\",\n  \"region\" : \"us-east-2\"\n }', signiture: '-----BEGIN PKCS7-----END PKCS7-----\n'};
+const no_ami = {document: '{\n  \"instanceId\" : \"i-aaaaaaaa\",\n  \"accountId\" : \"123456789012\",\n  \"imageId\" : \"ami-1qaz@WSX\",\n  \"region\" : \"us-east-2\"\n }', signiture: '-----BEGIN PKCS7-----END PKCS7-----\n'};
+const four_wrong = {document: '{\n  \"instanceId\" : \"im-aazaaaaa\",\n  \"accountId\" : \"123456z89012\",\n  \"imageId\" : \"i-abcdzaf2\",\n  \"region\" : \"uk-Northeast-Southwest-12\"\n}', signiture: '-----BEGIN PKCS7-----END PKCS7-----\n'};
```

Comment:
In addition to having an "everything is wrong" test case, you might want to have an "everything is right" test case too.